crypto/bio/bio_lib.c

```diff
@@ -61,10 +99,11 @@ int BIO_set(BIO *bio, const BIO_METHOD *method)
             CRYPTO_THREAD_lock_free(bio->lock);
             return 0;
         }
-    }
+    }   */
```

Comment:
Hmmmm....but I'm not sure this is better. We now have a way to create a BIO without calling its "create" function...which is also a "bad thing" and will probably also result in crashes/failures, i.e. imagine the scenario where BIO_new() is called with one BIO_METHOD, and then BIO_set() is called with a different BIO_METHOD. The create function for the first BIO_METHOD will have been called; destroy will not have been called; but everything else has been set up for the new BIO_METHOD. Here be dragons.